### DISCUSSION TITLE
Can't find the mol properties node

### DISCUSSION THREAD

**Discussion by nfloquet** [BradyAJohnston/MolecularNodes#232] [General]
Hello, incredible work !
just try the tutos but actually i'm unable to find the mol_prep_setup (attributes) node that is so useful to parse selections ...
Probably missed something ...
Where can i find this node ?

nicolas

**Comment by BradyAJohnston** [comment]
Hi @nfloquet, sorry about the confusion (as I do emphasise the node in the tutorials) but I have since updated the addon and the node is no longer needed. 

I haven't gotten around to updating the tutorials yet, but as long as you are seeing a protein in the 3D viewport then everything should be good, you don't need the `MOL_prop_setup` node anymore.

**Reply by nfloquet** [reply]
Thank you for your prompt answer ! be sure we will share some pictures/videos (if interested) made with it. Great job again !

**Reply by BradyAJohnston** [reply]
I look forward to seeing what you make with it :)

**Reply by nfloquet** [reply]
Just a last question, i try to add several representations on different parts of my system. How can i do that ? In one of the tutos you make "join geometry" but i can't achieve that. Do you have some screenshots showing typical nodes definition leading to a particular image ? would be useful to accelerate learning.

**Reply by BradyAJohnston** [reply]
The `Join Geometry` node still exists and that is how I still do it now, as in the tutorial example. You make the representations separately and join them together in for the final output.

**Reply by nfloquet** [reply]
found it ! thanks !

**Comment by smouilleron** [comment]
Hi Nicolas,
It's been a while! . I Hooe that things are going well for you!

If you start using Blender MN I advise you to join the dedicated  Discord channel.
https://discord.gg/snPYg9ZF
A great source of information!

Steph.




Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by nfloquet** [reply]
hey steph ! yes too long :-) Everything well here, I saw on linkedin it is pretty the same for your ! thank you for the info i will join the discord too !
